app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.kt

```diff
@@ -329,6 +330,7 @@ import javax.inject.Singleton
         AddVegetarian(),
         AddVegan(),
         AddKosher(),
+        AddHalal(),
```

Comment:
I think we should probably put Halal above Kosher. Based on my personal experience (I think I've seen more Halal than Kosher signage) and also https://en.wikipedia.org/wiki/List_of_religious_populations (there are 100 times more Muslims than Jews).
```suggestion
        AddHalal(), // There are many more Muslims than Jews
        AddKosher(),
```

Reply:
Did you not want to move this up @SMUsamaShah ?